### PR TITLE
Maptools - Make GPS display a real 3d object

### LINKED ISSUE #7051: request: On Map GPS movable
If I may make a request,

Please allow the on map GPS to be moved around.
It currently covers the bottom right of the map (the contour interval)

Otherwise great mod and great work.

### PR DESCRIPTION
**Pull request by PabstMirror**
Fixes #7051 (replaces PR #7052)

Makes the item gps display on the main map a real 3d model
drag-able and scale-able like the compass

example (with extra rotation applied, normally it just looks mostly 2d)

### CODE REVIEW

**Comment by jonpas** [comment]
That's pretty neat!

File: addons/maptools/functions/fnc_openMapGpsUpdate.sqf [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
- *
- * Return Value:
- * None
- *
- * Example:
- * call ACE_maptools_fnc_openMapGpsUpdate
- *
- * Public: No
- */
+* Author: esteldunedain, PabstMirror
```
Comment (mharis001) [inline addons/maptools/functions/fnc_openMapGpsUpdate.sqf]: Misaligned header.

File: addons/maptools/functions/fnc_openMapGpsUpdate.sqf [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
+* [findDisplay 12] call ACE_maptools_fnc_openMapGpsUpdate
+*
+* Public: No
+*/
 
-if ((!("ItemGPS" in assigneditems ACE_player)) || {isNull (uiNamespace getVariable [QGVAR(ui_mapGpsDisplay), displayNull])}) exitWith {
-    ("RscACE_MapGps" call BIS_fnc_rscLayer) cutText ["","PLAIN"];  // Close GPS RSC
-    [(_this select 1)] call CBA_fnc_removePerFrameHandler;  // Remove frameHandler
+params ["_mapCtrl"];
+private _mapDisplay = ctrlParent _mapCtrl;
```
Comment (commy2) [inline addons/maptools/functions/fnc_openMapGpsUpdate.sqf]: Function header says "map display" and example uses a DISPLAY, but `ctrlParent` only accepts CONTROL.

File: addons/maptools/functions/fnc_openMapGpsUpdate.sqf [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-* Example:
-* [findDisplay 12] call ACE_maptools_fnc_openMapGpsUpdate
-*
-* Public: No
-*/
+ * Author: esteldunedain, PabstMirror
+ * update gps display, called from main map's draw
+ *
+ * Arguments:
+ * 0: Map ctrl <DISPLAY>
```
Comment (commy2) [inline addons/maptools/functions/fnc_openMapGpsUpdate.sqf]:
```suggestion
 * 0: Map ctrl <CONTROL>
```

File: addons/maptools/functions/fnc_openMapGpsUpdate.sqf [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [findDisplay 12] call ACE_maptools_fnc_openMapGpsUpdate
```
Comment (commy2) [inline addons/maptools/functions/fnc_openMapGpsUpdate.sqf]:
```suggestion
 * [findDisplay 12 displayCtrl 51] call ACE_maptools_fnc_openMapGpsUpdate;
```

**Pull request merged**